egs/arctic/vc/local/ob_eval/evaluate.sh

```diff
@@ -0,0 +1,152 @@
+#!/bin/bash
+
+# Copyright 2020 Nagoya University (Wen-Chin Huang)
+#  Apache 2.0  (http://www.apache.org/licenses/LICENSE-2.0)
+
+# Evaluation script for VC
+
+
+echo "$0 $*"  # Print the command line for logging
+. ./path.sh
+
+nj=1
+do_delta=false
+db_root=""
+backend=pytorch
+api=v2
+vocoder=
+mcep_dim=24
+shift_ms=5
+asr_model="librispeech.transformer.ngpu4"
+help_message="Usage: $0 <outdir> <subset> <srcspk> <trgspk>"
+
+. utils/parse_options.sh
+
+outdir=$1
+set_name=$2  # <srcspk>_<trgspk>_<name> 
+
+if [ $# != 2 ]; then
+    echo "${help_message}"
+    exit 1;
+fi
+
+set -euo pipefail
+
+# parse srcspk, trgspk and name
+srcspk=$(echo ${set_name} | awk -F"_" '{print $1}')
+trgspk=$(echo ${set_name} | awk -F"_" '{print $2}')
+name=$(echo ${set_name} | awk -F"_" '{print $3}')
+    
+# Decide wavdir depending on vocoder
+if [ ! -z ${vocoder} ]; then
+    # select vocoder type (GL, PWG)
+    if [ ${vocoder} == "PWG" ]; then
+        wavdir=${outdir}_denorm/${set_name}/pwg_wav
+    elif [ ${vocoder} == "GL" ]; then
+        wavdir=${outdir}_denorm/${set_name}/wav
+    else
+        echo "Vocoder type other than GL, PWG is not supported!"
+        exit 1
+    fi
+else
+    echo "Please specify vocoder."
+    exit 1
+fi
+
+echo "MCD calculation"
+mcd_file=${outdir}_denorm/${set_name}/mcd.log
+minf0=$(awk '{print $1}' conf/${trgspk}.f0)
+maxf0=$(awk '{print $2}' conf/${trgspk}.f0)
+${decode_cmd} ${mcd_file} \
+    mcd_calculate.py \
+        --wavdir ${wavdir} \
+        --gtwavdir ${db_root}/cmu_us_${trgspk}_arctic/wav \
+        --mcep_dim ${mcep_dim} \
+        --shiftms ${shift_ms} \
+        --f0min ${minf0} \
+        --f0max ${maxf0}
```

Comment:
I have a general question about MCD.
In the case of world-based mcep, the MCD value is affected by the f0 prediction.
So I think the use of pysptk-based mcep is more robust.
How do you think?

Reply:
I have never calculated and compared pysptk-based mcep, but I have been calculated this way in my VTN papers, so it might be better to keep the reproducibility?

Reply:
OK.